docs/design/2019-04-11-indexmerge.md

```diff
@@ -1,4 +1,4 @@
-# Proposal: scan a table using IndexMerge

```

Comment:
This file seems to have two types of line endings, i.e, linux LF and windows CRLF, which would cause annoying changes in git workspace. Check this behavior on linux out:
```
[root@localhost ~/go/src/github.com/pingcap/tidb]$ git status
# On branch cost_model
# Untracked files:
#   (use "git add <file>..." to include in what will be committed)
#
#	cscope.files
#	cscope.out
nothing added to commit but untracked files present (use "git add" to track)
[root@localhost ~/go/src/github.com/pingcap/tidb]$ git checkout -t origin/master
Branch master set up to track remote branch master from origin.
Switched to a new branch 'master'
[root@localhost ~/go/src/github.com/pingcap/tidb]$ git status
# On branch master
# Changes not staged for commit:
#   (use "git add <file>..." to update what will be committed)
#   (use "git checkout -- <file>..." to discard changes in working directory)
#
#	modified:   docs/design/2019-04-11-indexmerge.md
#
# Untracked files:
#   (use "git add <file>..." to include in what will be committed)
#
#	cscope.files
#	cscope.out
no changes added to commit (use "git add" and/or "git commit -a")
```
Please resolve this in this PR as well.

Reply:
ok